server.go

```diff
@@ -540,6 +540,20 @@ func (s *Server) errorf(format string, a ...interface{}) {
 	}
 }
 
+// ServiceRegistrar wraps a single method that supports service registration. It
+// enables users to pass concrete types other than grpc.Server to the service
+// registration methods exported by the IDL generated code.
+//
+// This API is EXPERIMENTAL.
```

Comment:
This can't be experimental if we use it in stable generated code.

Reply:
Hmm ....
Done.